tests/functional/firefox/test_accounts.py

```diff
@@ -7,6 +7,7 @@
 from pages.firefox.accounts import FirefoxAccountsPage
 
 
+@pytest.mark.skipif(reason='Page is temporarily behind the switch "firefox_accounts_trailhead"')
```

Comment:
I believe you need to use `skip()` here. `skipif()` takes another argument that is a conditional.

```suggestion
@pytest.mark.skip(reason='Page is temporarily behind the switch "firefox_accounts_trailhead"')
```